lib/data/errors/failure.dart

```diff
@@ -0,0 +1,5 @@
+import 'package:superformula_test/core/resources/failure.dart';
+
+class RepositoryFailure extends Failure {
```

Comment:
suggestion: this could benefit from const constructors too if `Failure` had a constant constructor